deps-tasks.yml

```diff
@@ -421,8 +421,8 @@ tasks:
         vars:
           DEST: "{{.DEST}}"
           FLAGS: "--extract"
-          SRC_NAME: "yscope-log-viewer-df996eac000823d02f9cd0b9eb4bb732dd634ae5"
-          SRC_URL: "https://github.com/y-scope/yscope-log-viewer/archive/df996ea.zip"
+          SRC_NAME: "yscope-log-viewer-c939f7b55b55b42f65226470d5277b15ac484665"
```

Comment:
Can you also update the submodule in components/log-viewer-webui/yscope-log-viewer? See https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt---remote

Reply:
Oh, I thought I shouldn't push the updated package. Let me fix it

Reply:
Not sure if I get what's the "updated package" but here're the steps based on how I would normally change the submodule version in a repo:
```
# not validated; can't guarantee below works
cd components/log-viewer-webui/yscope-log-viewer

# i believe `git fetch` works the same here
git pull

# here c939f7b55b55b42f65226470d5277b15ac484665 is the latest commit id
git checkout c939f7b55b55b42f65226470d5277b15ac484665
```

Note the tasks in `dep-tasks.yml` are meant to download the submodule (nested git repo) files when git cloning with submodules recursively is impossible / inconvenient.

Reply:
Not sure if I get what's the "updated package" but here're the steps based on how I would normally change the submodule version in a repo:
```
# not validated; can't guarantee below works
cd components/log-viewer-webui/yscope-log-viewer

# i believe `git fetch` works the same here
git pull

# here c939f7b55b55b42f65226470d5277b15ac484665 is the latest commit id
git checkout c939f7b55b55b42f65226470d5277b15ac484665
```

Note the tasks in `dep-tasks.yml` are meant to download the submodule (nested git repo) files when git cloning with submodules recursively is impossible / inconvenient.